pkg/reconciler/pipelinerun/resources/pipelinerunresolution.go

```diff
@@ -504,6 +506,19 @@ func (t *ResolvedPipelineTask) skipBecausePipelineRunFinallyTimeoutReached(facts
 	return false
 }
 
+// skipBecauseEmptyArrayInMatrixParams returns true if the matrix parameters contain an empty array
+func (t *ResolvedPipelineTask) skipBecauseEmptyArrayInMatrixParams(facts *PipelineRunFacts) bool {
```

Comment:
Is the `facts` used in this function? Maybe we can remove it if not used.

Reply:
Yes, good catch. Thanks!